app/controllers/themes_controller.rb

```diff
@@ -25,7 +25,6 @@ def new
   def create
     @theme = Theme.new(theme_params)
     @theme.user = current_user
-    @theme.price_cent = @theme.price_cent * 100
```

Comment:
Have you confirmed that, if we remove this line, when we create new theme, the price will not be impacted?

Reply:
All is working, now the mathematic calculate is done at the same place. I tested creating and updating.

Reply:
nice, will merge it now

Reply:
(price_cente *100) /100